src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1458,6 +1458,14 @@ public override IOperation VisitBinaryOperator(IBinaryOperation operation, int?
                                                 operation.OperatorMethod, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
         }
 
+        public override IOperation VisitTupleBinaryOperator(ITupleBinaryOperation operation, int? captureIdForResult)
+        {
+            _evalStack.Push(Visit(operation.LeftOperand));
```

Comment:
>_evalStack.Push(Visit(operation.LeftOperand)); [](start = 12, length = 46)

If operands are tuple expressions, should we preserve those as we do for deconstruction, for example?

Reply:
If we do that, we probably should do that for the other side as well, but it looks like we are not doing this for deconstruction. Probably should do as well.

---
In reply to: [189724034](https://github.com/dotnet/roslyn/pull/26919#discussion_r189724034) [](ancestors = 189724034)